packages/web-app/src/conf/config.js

```diff
@@ -91,3 +91,5 @@ export const NOMINATIM_API_URL = 'https://nominatim.openstreetmap.org/search';
 
 export const AUTOCOMPLETE_DEBOUNCE_DELAY = 500; // in milliseconds
 export const AUTOCOMPLETE_MIN_CHARACTERS = 2;
+
+export const DEFAULT_PAGE_SIZE_OPTIONS = [20, 100, 200];
```

Comment:
As this is not a global option may it can be put inside the `EntityTable.jsx‎` file alongside `MAX_DOCUMENTS_TO_EXPORT_IN_CSV`